src/components/Bookitem.js

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Bookitem = (props) => {
-  const { title, author, deleteBook } = props;
+  const { title, author, remove } = props;
 
   return (
     <div className="container">
       <div className="bookCard">
         <p className="title">{title}</p>
         <p className="author">{author}</p>
-        <button type="button" className="Remove" onClick={deleteBook}>Remove</button>
+        <button type="button" className="Remove" onClick={remove}>Remove</button>
```

Comment:
- Some of the[ requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#project-requirements) below are not yet implemented in this component ⬇️ 

```
In your React component responsible for removing books, add a functionality for "Remove" button click:
Import Redux useDispatch.
Dispatch a corresponding action (import Action Creator that you created before and use it here).
```

- [ ]  Kindly make sure that you import the action creator in your redux  and use it in creating the function for removing the book 👍🏿